frontends/web/src/api/bitbox02.ts

```diff
@@ -68,6 +68,14 @@ export type VersionInfo = {
     currentVersion: string;
     canUpgrade: boolean;
     canGotoStartupSettings: boolean;
```

Comment:
@thisconnect how come our linter forces 2 spaces everywhere but not inside type declarations? :see_no_evil:

Reply:
I think because eslint doesn't know typescript.